build.gradle.kts

```diff
@@ -102,7 +102,7 @@ apply(from = "gradle/compatibility.gradle")
 plugins {
     id("org.jetbrains.dokka") version "1.9.20" apply false
     id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.16.3"
-    id("org.jetbrains.kotlinx.atomicfu") version "0.25.0" apply false
+    //id("org.jetbrains.kotlinx.atomicfu") version "0.25.0" apply false
```

Comment:
What was the original purpose of disabling the plugin here? 👀

Reply:
I think the purpose is to add the plugin to the build classpath but apply it later